site/webpack.mix.js

```diff
@@ -0,0 +1,51 @@
+let mix = require('laravel-mix');
+let argv = require('yargs').argv;
+let bin = require('./tasks/bin');
+let WebpackShellPlugin = require('webpack-shell-plugin');
+
+let env = argv.e || argv.env || 'local';
+var port = argv.p || argv.port || 3000;
```

Comment:
@afontcu You are right, but I mostly tried to stay consistent with the Mix Codestyle which is using let in these cases as well. 

Really seems to be the Box of Pandora 😄